net.sourceforge.uqm_mods.UQM-MegaMod.appdata.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>net.sourceforge.uqm_mods.UQM-MegaMod</id>
+    <name>The Ur-Quan Masters HD MegaMod</name>
+    <launchable type="desktop-id">net.sourceforge.uqm_mods.UQM-MegaMod.desktop</launchable>
+    <summary>A mod for The Ur-Quan Masters</summary>
```

Comment:
```suggestion
    <summary>Mod for The Ur-Quan Masters</summary>
```

Reply:
https://docs.flathub.org/docs/for-app-authors/appdata-guidelines/quality-guidelines#dont-start-with-an-article